src/list-grid.scss

```diff
@@ -0,0 +1,232 @@
+@import './new-settings';
+@import './mixins';
+
+/*!
+.fd-grid-list
+*/
+$block: #{$fd-namespace}-grid-list;
+
+.#{$block} {
+  $fd-grid-list-border-radius: var(--sapElement_BorderCornerRadius);
+  $fd-grid-list-active-text-color: var(--sapList_Active_TextColor);
+  $fd-grid-list-item-box-shadow: var(--sapContent_Shadow0);
+  $fd-grid-list-mode-box-shadow: 0 0 0 0.125rem var(--sapList_SelectionBorderColor);
+  $fd-grid-list-highlight: (
+    'success': ('backgroundColor': var(--sapSuccessBorderColor)),
+    'error': ('backgroundColor': var(--sapErrorBorderColor)),
+    'warning': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+
+  display: flex;
+  flex-direction: column;
+
+  .fd-list {
+    &__item {
+      border-radius: $fd-grid-list-border-radius;
+      box-shadow: $fd-grid-list-item-box-shadow;
+      border: 0 !important;
+      padding: 0 !important;
+      flex-direction: column;
+      align-items: normal !important;
+      height: 100% !important;
+
+      .fd-toolbar {
+        padding: 0 1rem;
+        align-items: center;
+      }
+
+      &-body {
+        padding: 1rem;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+      }
+
+      &--counter {
+        color: var(--sapContent_MarkerTextColor);
+      }
+
+      .fd-checkbox + .fd-checkbox__label,
+      .fd-radio + .fd-radio__label {
+        display: none;
+        padding: 0 !important;
+        margin: 0 !important;
+      }
+
+      .fd-radio + .fd-radio__label::after {
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        margin: auto !important;
+      }
+
+      &--state-unread {
+        .fd-list__item-body * {
+          font-weight: bold;
+        }
+      }
+
+      &--state-actions {
+        margin-top: 1rem;
+      }
+
+      &--link {
+        @include fd-navigated() {
+          &::after {
+            content: "";
+            position: absolute;
+            top: 0;
+            right: 0;
+            height: 100%;
+            width: 0.1875rem;
+            display: block;
+            border-radius: 0 $fd-grid-list-border-radius $fd-grid-list-border-radius 0;
+            background-color: var(--sapList_SelectionBorderColor);
+          }
+        }
+      }
+    }
+
+    &__link {
+      flex-direction: column;
+      align-items: normal;
+      padding: 0;
+
+      @include fd-active() {
+        * {
+          color: $fd-grid-list-active-text-color !important;
+        }
```

Comment:
That's not a good idea :)

Reply:
Do you know another way to avoid using * if we need to apply color for all inside an item and namely inside the body where we do not know what will be inside?
I will be glad for your help @InnaAtanasova :slightly_smiling_face: